detection_rules/integrations.py

```diff
@@ -0,0 +1,127 @@
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License
+# 2.0; you may not use this file except in compliance with the Elastic License
+# 2.0.
+
+"""Integration functions."""
+import gzip
+import json
+import os
+import re
+
+import yaml
+
+from .ghwrap import GithubClient
+from .semver import Version
+from .utils import INTEGRATION_RULE_DIR, get_etc_path
+
+
+class IntegrationPackages():
```

Comment:
looks like there is no reason for this class and that you can just convert the methods to standalone functions